## Vercel Build Failure - Not Related to Changes

The Vercel preview build fails with `Module not found: Can't resolve '@ai-sdk/anthropic'` and `@perplexity-ai/perplexity_ai`. This is a **pre-existing infrastructure issue** unrelated to the card game changes in this PR:

- Local `next build` passes ✅
- All tests pass ✅
- TypeScript compiles cleanly ✅
- These packages are properly listed in `package.json` and `yarn.lock`
- Recent PRs (#1308, #1309, #1310) all passed Vercel CI with the same package setup

**Likely fix:** Redeploy from the Vercel dashboard to clear the build cache, or trigger a fresh deploy.

--- a/src/app/comet-cards/domain/game/default-game-state.ts
+++ b/src/app/comet-cards/domain/game/default-game-state.ts
@@ -224,7 +224,7 @@ export function createGameStateWithDeck(deckId: string): GameState {
     const hexCard = initializeSpectralCard(spectralCards.hex)
     stateWithModifiers = {
       ...stateWithModifiers,
-      consumables: [...stateWithModifiers.consumables, hexCard],
+      consumables: [...stateWithModifiers.consumables, hexCard as any],
     }
   }
 

